### PR TITLE
Changelevel

### PR DESCRIPTION
**Pull request by CarGuerrero**

### CODE REVIEW

File: Source/RN4UE4/RN4UE4GameInstance.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+{
+	GENERATED_BODY()
+	
+public:
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Levels")
+		int indexLevel;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Levels")
+		TArray<FName> levelNames;
```
Comment (Alexthegr92) [inline Source/RN4UE4/RN4UE4GameInstance.h]: This can be removed

File: Source/RN4UE4/RN4UE4GameInstance.h [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ */
+UCLASS()
+class RN4UE4_API URN4UE4GameInstance : public UGameInstance
+{
+	GENERATED_BODY()
+	
+public:
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Levels")
+		int indexLevel;
```
Comment (Alexthegr92) [inline Source/RN4UE4/RN4UE4GameInstance.h]: This can be removed

File: Source/RN4UE4/RakNetRP.cpp [hunk 3]
```diff
@@ -128,6 +128,7 @@ void ARakNetRP::Tick(float DeltaTime)
 			allServersChecked = true;
 		}
 	}
+
```
Comment (Alexthegr92) [inline Source/RN4UE4/RakNetRP.cpp]: Undo this change

File: Config/DefaultEngine.ini [hunk 4]
```diff
@@ -43,4 +44,3 @@ AsyncSceneSmoothingFactor=0.990000
 InitialAverageFrameRate=0.016667
 PhysXTreeRebuildRate=10
 
-
```
Comment (Alexthegr92) [inline Config/DefaultEngine.ini]: Undo this change

File: Source/RN4UE4/RN4UE4GameInstance.h (hunk 2 above)
Comment (CarGuerrero) [reply Source/RN4UE4/RN4UE4GameInstance.h]: I use that variable to check what level have been loaded to persistent level

**Review by Alexthegr92 — APPROVED** [review]
